src/lsp/cobol_ptree/proc_division_visitor.ml

```diff
@@ -132,12 +135,26 @@ let fold_raising_phrase (v: _ #folder) =
 let fold_raising_phrase' (v: _ #folder) =
   handle' v#fold_raising_phrase' ~fold:fold_raising_phrase v
 
+let fold_procedure_calling_style (v: _ #folder) =
+  leaf v#fold_procedure_calling_style
+
+let fold_procedure_calling_style' (v: _ #folder) =
+  leaf' v#fold_procedure_calling_style'
+
+let fold_procedure_args (v: _ #folder) =
+  handle v#fold_procedure_args
+    ~continue:begin fun
+      {procedure_calling_style=style; procedure_by_clause=args} x -> x 
```

Comment:
Please follow the overall style, in particular for visitors: with the spaces after/before `{` and `}`, and around `=` (and no line exceeding 80 chars appox.):
```ocaml
    ~continue:begin fun { procedure_calling_style = style;
                          procedure_by_clause = args } x -> x 
   …
```